setup.cfg

```diff
@@ -26,6 +26,7 @@ package_dir =
 include_package_data = True
 install_requires =
     astropy>=2.0
+    astropy>=3.2
```

Comment:
Astropy 2.0 is the last version that supports python 2, so this is impossible to satisfy on python 2. You can put a python_version constraint on this, as I did for sphinx, but I thought I put this constraint into master?

Reply:
I edited setup.cfg based on the nanograv/master version of setup.cfg, I think it matches?

Reply:
Merging should not be too difficult, that's probably the best way to get new changes pulled in.

If the upstream changes are small, or your changes are small, merging isn't usually much of a challenge unless you're changing the same thing that upstream is changing. It's also easier if you can run the test suite, or even just parts of it, to check your merge doesn't break anything new.

On the other hand, if you make the same changes upstream does, merging may or may not recognize that you both made the same changes, and you can end up having to sort out a lot of messy merge conflicts.

Reply:
Makes sense, thanks for the advice! I'm not very familiar with github so I'm never entirely sure what the best practices are. I'll merge from now on.

Reply:
I'm afraid you are having something of a baptism of fire - it is much easier to handle all these things with a small localized change but you have a large extensive set of changes to merge, which amplifies all the challenges. 

If there's some small standalone change - adding TOA generation maybe? something smallish that doesn't depend on anything else - it would be easier to merge it into mainline PINT, but then it gets a little complicated merging the rest of your changes without conflicting with the pieces already included. If you want to try something like that, let me know and I'll see if I can help walk you through the process.